scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/security/Protocols.kt

```diff
@@ -0,0 +1,9 @@
+package io.seldon.dataflow.kafka.security
```

Comment:
:wrench: We need to re-run licence header generation

Reply:
Which target is that?

Reply:
`make update-copyright` I think, at the top level